src/ray/core_worker/task_manager.cc

```diff
@@ -10,11 +10,10 @@ const int64_t kTaskFailureThrottlingThreshold = 50;
 // Throttle task failure logs to once this interval.
 const int64_t kTaskFailureLoggingFrequencyMillis = 5000;
 
-void TaskManager::AddPendingTask(const TaskID &caller_id,
+void TaskManager::AddPendingTaskImpl(const TaskID &caller_id,
```

Comment:
I think in other places in the code (e.g., in `reference_count.h`) we call similar methods `XyzInternal`. No strong opinion but would be good to be consistent.